todoman/cli.py

```diff
@@ -48,6 +48,23 @@ def _validate_date_param(ctx, param, val):
         raise click.BadParameter(e)
 
 
+def _validate_start_date_param(ctx, param, val):
+    try:
```

Comment:
Only the invocation of `parse_date` has to be in the try-except block. Moving the rest out of it spares unnecessary indentation.